### DISCUSSION TITLE
4.1 release?

### DISCUSSION THREAD

**Discussion by lfcnassif** [sepinf-inc/IPED#1491] [General]
All devs,

I'm planning to roll a 4.1 release soon, there are many relevant new features finished and merged like #306 and #1214. I still want to include #1286. Should we include something else?

**Comment by wladimirleite** [comment]
Hi @lfcnassif!
I would like to include:
#1474: A simple change, I should finish it soon.
#1461: So far I only downloaded the new NRSL files and thought about the implementation, but didn't start coding it.
#1344: Spent quite a lot of time on this. Running tests on real cases, I got some interesting results, but also found some critical issues. Still need to figure out a few details. And the the review process should take some time, as I had a few "question marks" during the development process. So depending on how soon you plan to release 4.1, I will try to finish the pending issues or leave it for the next realease.

**Reply by lfcnassif** [reply]
Hi @tc-wleite. I was hoping to release on Friday, but probably it will be on the next week. Another one that could be included is #1397...

I've just updated the release notes with the merged changes:
https://github.com/sepinf-inc/IPED/blob/master/ReleaseNotes.txt

**Reply by wladimirleite** [reply]
> Hi @tc-wleite. I was hoping to release on Friday, but probably it will be on the next week. Another one that could be included is #1397...

Ok! I will try to submit a PR with #1474 tomorrow. Probably I will leave the other two for a future release.

**Reply by lfcnassif** [reply]
#1344 would be great! But I don't know how much effort it still needs, don't worry and take your time. I can help testing of course.

**Reply by wladimirleite** [reply]
I am finishing #1461. 
After running more tests, I will submit a PR, probably tomorrow. 
As NIST announced that _"The new RDSv3 format will be the only publication format released by the NSRL beginning March 2023"_, it would be great to include this in 4.1.0.

**Reply by lfcnassif** [reply]
Thank you @tc-wleite! But don't hurry, probably I'll need some days to finish reviewing and testing other PRs.

**Reply by lfcnassif** [reply]
I just updated 4.1 roadmap with more features I think would be good to include: https://github.com/sepinf-inc/IPED/projects/2

**Reply by lfcnassif** [reply]
#687 is virtually finished. I'll send the new localization strings for the kind foreign contributors helping with translations.

@tc-wleite do you think #1340 would be worth to include? If we have luck, it should take 1-2 days.

**Reply by wladimirleite** [reply]
> @tc-wleite do you think #1340 would be worth to include? If we have luck, it should take 1-2 days.

I am not sure. It seems an important upgrade, but I guess it may take a while to integrate and test.

**Reply by lfcnassif** [reply]
Ok I already asked for the localization updates to our foreign collaborators. If they are fast, we can leave TSK-4.12 for another release. If they need some time, maybe it could be included.
